apps/music/test/marionette/Player_test.js

```diff
@@ -133,4 +133,39 @@ marionette('Music player tests', function() {
       music.checkPlayerIconShown(true);
     });
   });
+
+  suite('Rating test', function() {
+    test('Rating persistance', function() {
+      music.launch();
+      music.waitForFirstTile();
+      music.switchToSongsView();
+      music.playFirstSong();
+
+      var rating_value = 4;
```

Comment:
Rather than hard-coding a single value, how putting a loop here and testing all 5 stars to make sure they work?  Would that make sense?